app/src/components/blocks/_lloydGeorge/lloydGeorgeViewRecordStage/LloydGeorgeViewRecordStage.test.tsx

```diff
@@ -53,13 +53,13 @@ describe('LloydGeorgeViewRecordStage', () => {
     });
 
     it('renders an lg record', async () => {
+        mockedUseRole.mockReturnValue(REPOSITORY_ROLE.GP_ADMIN);
```

Comment:
instead, can we assert that the view in full screen button is not visible for GP_clinical role

Reply:
I realise that after having aligned the code to main to resolve merge conflicts, this GP_ADMIN mock is no longer needed. I have removed the line and the test is passing fine.